### PR TITLE
Converting portable pdbs to full pdbs for Symbol Archiving

### PR DESCRIPTION
**Pull request by jayaranigarg**

### CODE REVIEW

File: scripts/PortableToFullPdb.ps1 [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# Portable to Full PDB conversion script for Test Platform.
+
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$false)]
+    [ValidateSet("Debug", "Release")]
+    [System.String] $Configuration = "Release"
+)
+
+$ErrorActionPreference = "Stop"
```
Comment (smadala) [inline scripts/PortableToFullPdb.ps1]: No usage of it.

File: NuGet.config [hunk 2]
```diff
@@ -12,6 +12,7 @@
     <add key="nuint.myget" value="https://www.myget.org/F/nunit/api/v3/index.json" />
     <add key="xunit.myget" value="https://www.myget.org/F/xunit/api/v3/index.json" />
     <add key="mstest.myget" value="https://dotnet.myget.org/F/mstestv2/api/v3/index.json"/>
+	<add key="pdb2pdb.myget" value="https://dotnet.myget.org/F/symreader-converter/api/v3/index.json" />
```
Comment (smadala) [inline NuGet.config]: Nit:

File: scripts/PortableToFullPdb.ps1 [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    {
+		# First check if corresponding dll exists
+        $dllOrExePath = $portablePdb -replace ".pdb",".dll"
+		
+		if(!(Test-Path -path $dllOrExePath))
+		{
+			# If no corresponding dll found, check if exe exists
+			$dllOrExePath = $portablePdb -replace ".pdb",".exe"
+			
+			if(!(Test-Path -path $dllOrExePath))
```
Comment (smadala) [inline scripts/PortableToFullPdb.ps1]: Nit: indent.

**Review by smadala — APPROVED** [review]

**Comment by jayaranigarg** [comment]
@dotnet-bot test Windows_NT / Release Build please

**Pull request merged**